packages/react/src/themes/teams/components/Icon/svg/icons/image-unavailable.tsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react'
+import { TeamsSvgIconSpec } from '../types'
+
+export default {
+  icon: ({ classes }) => (
+    <svg role="presentation" focusable="false" viewBox="0 0 32 32" className={classes.svg}>
```

Comment:
Is this viewbox correct? Other icons have something like 8 8 16 16.

Reply:
Yes, that's the way the svg was created. I have verified that the icon looks correctly compared with other icons. For example, I compared it with "highlight" which has a viewbox="8 8 16 16" across the different supported sizes.

![image](https://user-images.githubusercontent.com/21109913/61478716-f8e3e300-a946-11e9-8a0d-9e6f3515adab.png)

Reply:
Spoke offline with @notandrew and @codepretty and this particular svg doesn't have a padding within the image, hence it is okay to keep viewport="0 0 32 32"